packages/styles/src/token.scss

```diff
@@ -3,58 +3,64 @@
 
 $block: #{$fd-namespace}-token;
 
+@mixin fd-readonly-token {
+  --fdToken_Text_Color: var(--sapContent_LabelColor);
+  --fdToken_Padding_Inline: var(--fdToken_ReadOnly_Padding_Inline);
+  --fdToken_Border: var(--sapButton_BorderWidth) solid var(--fdToken_Border_Color_Read_Only);
+
+  background-color: var(--fdToken_Background_Color_Read_Only);
+
+  @include fd-compact-or-condensed() {
+    --fdToken_Padding_Inline: 0.25rem;
+  }
+
+  @include fd-hover() {
+    --fdToken_Text_Color: var(--sapContent_LabelColor);
+    --fdToken_Border: var(--sapButton_BorderWidth) solid var(--fdToken_Border_Color_Read_Only);
+
+    background-color: var(--fdToken_Background_Color_Read_Only);
+  }
+}
+
 .#{$block} {
+  --fdToken_Background_Color: var(--sapButton_TokenBackground);
+  --fdToken_Border: var(--sapButton_BorderWidth) solid var(--sapButton_TokenBorderColor);
+  --fdToken_Padding_Inline: 0.3125rem 0;
+  --fdToken_Height: 1.625rem;
+  --fdToken_Min_Width: 3.625rem;
+  --fdToken_Close_Color: var(--sapContent_IconColor);
+  --fdToken_Close_Padding: 0.25rem 0.5rem;
+  --fdToken_Close_Width: 1.75rem;
+  --fdToken_Close_Compact_Width: 1.25rem;
```

Comment:
`--fdToken_Close_Compact_Width` doesn't have to be defined here. 
On line 113 just do `--fdToken_Close_Width: 1.25rem`. 
Will be easier to maintain.